architecture/service_catalog/aws_broker.adoc

```diff
@@ -0,0 +1,70 @@
+[[admin-guide-get-started-aws]]
+= {product-title} Amazon Web Services Broker
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+The _AWS Broker_ provides access to Amazon Web Services (AWS) through the {product-title} Service Catalog. 
+The AWS Broker is {product-title}'s implementation of the link:https://aws.amazon.com/partners/servicebroker/[AWS Service Broker]. 
```

Comment:
Is there a reason we are calling this the 'AWS Broker' and saying it is an implementation of the 'AWS Service Broker'.  This feels strange to me.

I think of this as just the 'AWS Service Broker', I don't follow the distinction of why we'd say this is the 'AWS Broker' implementing 'AWS Service Broker'.

The 'AWS Service Broker' is a Service Broker written/delivered by Amazon that implements the Open Service Broker API, publishing AWS Services to the Service Catalog.

Reply:
@jwmatthews The AWS Broker wording was in the original upstream draft [1], which led me to assume it was our tool based on the AWS tool; . I will fix this. 

[1] https://github.com/awslabs/aws-servicebroker/commit/1c6fcabda3804a83e8a1c7b94a806947f5b5272d#diff-0a451d7312f851709884f550564b731d